Starkcore/utils/Request.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Newtonsoft.Json.Linq;
+using System.Globalization;
+using EllipticCurve;
+using StarkCore;
+using StarkCore.Error;
+using StarkCore.Utils;
+using System.Security.Cryptography;
+using System.Xml.Linq;
+
+
+namespace StarkCore.Utils
+{
+    internal class Response
+    {
+        internal byte[] ByteContent { get; }
+        internal int Status { get; }
+
+        internal Response(byte[] byteContent, int status)
+        {
+            ByteContent = byteContent;
+            Status = status;
+        }
+
+        internal string Content
+        {
+            get
+            {
+                return System.Text.Encoding.UTF8.GetString(ByteContent);
+            }
+        }
+
+        internal JObject Json()
+        {
+            return Utils.Json.Decode(Content);
+        }
+    }
+
+    internal static class Request
+    {
+        private static HttpClient makeClient()
+        {
+            HttpClient client = new HttpClient();
+            return client;
+        }
+
+        private static readonly HttpClient Client = makeClient();
+        internal static readonly HttpMethod Get = new HttpMethod("GET");
+        internal static readonly HttpMethod Put = new HttpMethod("PUT");
+        internal static readonly HttpMethod Post = new HttpMethod("POST");
+        internal static readonly HttpMethod Patch = new HttpMethod("PATCH");
+        internal static readonly HttpMethod Delete = new HttpMethod("DELETE");
+
+        internal static Response Fetch(User user, HttpMethod method, string path, string host, string apiVersion, string sdkVersion, Dictionary<string, object> payload = null, Dictionary<string, object> query = null)
+        {
+            user = Checks.CheckUser(user);
+
+            string url = "";
+            if (user.Environment == "production")
+            {
+                url = "https://api.stark" + host + ".com/";
```

Comment:
Caso mude para enum, tem que ver como fica a variável host aqui. Se for só manter o que já estava, aqui esta ok